### DISCUSSION TITLE
What does the score output by `retrieve` mean？

### DISCUSSION THREAD

**Discussion by Ask-sola** [xhluca/bm25s#65] [Q&A] [answered]
What is an appropriate normalization method for normalizing this score?

**Comment by Ask-sola** [comment]
Most importantly, is the scoring standard affected by the query itself (e.g., longer sentences scoring higher overall, shorter sentences scoring lower overall)?

**Comment by xhluca** [comment]
If you are are asking about how BM25 score works, i highly recommend reading the review by one of the original authors: https://www.staff.city.ac.uk/~sbrp622/papers/foundations_bm25_review.pdf

**Reply by Ask-sola** [reply, marked as answer]
Thank you. In fact, what I want is to set a threshold such that for a query, if the score is higher than the threshold, the two are highly likely to be related. However, I have noticed that for different queries, even when the content is relevant to the query, the score can differ by several orders of magnitude.

**Reply by xhluca** [reply]
That's interesting! NOt totally sure why it's happening, likely due to how the score is computed. feel free to try different score
